docs/src/man/basics.md

```diff
@@ -0,0 +1,1220 @@
+    8 │ male        3  rent
+  ⋮   │   ⋮       ⋮       ⋮
+  994 │ male        3  own
+  995 │ male        2  own
+  996 │ female      1  own
+  997 │ male        3  own
+  998 │ male        2  own
+  999 │ male        2  free
+ 1000 │ male        2  own
+               985 rows omitted
+
+julia> german[:, Cols("Age", Between("Sex", "Job"))]
+1000×3 DataFrame
+  Row │ Age    Sex     Job
+      │ Int64  String  Int64
+──────┼──────────────────────
+    1 │    67  male        2
+    2 │    22  female      2
+    3 │    49  male        1
+    4 │    45  male        2
+    5 │    53  male        2
+    6 │    35  male        1
+    7 │    53  male        2
+    8 │    35  male        3
+  ⋮   │   ⋮      ⋮       ⋮
+  994 │    30  male        3
+  995 │    50  male        2
+  996 │    31  female      1
+  997 │    40  male        3
+  998 │    38  male        2
+  999 │    23  male        2
+ 1000 │    27  male        2
+             985 rows omitted
+
+julia> german[:, Cols("Age", Not("Sex"))]
+1000×9 DataFrame
+  Row │ Age    id     Job    Housing  Saving accounts  Checking account  Credi ⋯
+      │ Int64  Int64  Int64  String   String           String            Int64 ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │    67      0      2  own      NA               little                  ⋯
+    2 │    22      1      2  own      little           moderate
+    3 │    49      2      1  own      little           NA
+    4 │    45      3      2  free     little           little
+    5 │    53      4      2  free     little           little                  ⋯
+    6 │    35      5      1  free     NA               NA
+    7 │    53      6      2  own      quite rich       NA
+    8 │    35      7      3  rent     little           moderate
+  ⋮   │   ⋮      ⋮      ⋮       ⋮            ⋮                ⋮                ⋱
+  994 │    30    993      3  own      little           little                  ⋯
+  995 │    50    994      2  own      NA               NA
+  996 │    31    995      1  own      little           NA
+  997 │    40    996      3  own      little           little
+  998 │    38    997      2  own      little           NA                      ⋯
+  999 │    23    998      2  free     little           little
+ 1000 │    27    999      2  own      moderate         moderate
+                                                  3 columns and 985 rows omitted
+```
+
+You can also use `Regex` to select columns and `Not` from `InvertedIndices.jl` both to select rows and columns:
+```jldoctest dataframe
+julia> german[Not(5), r"Sex"]
+999×1 DataFrame
+ Row │ Sex
+     │ String
+─────┼────────
+   1 │ male
+   2 │ female
+   3 │ male
+   4 │ male
+   5 │ male
+   6 │ male
+   7 │ male
+   8 │ male
+  ⋮  │   ⋮
+ 993 │ male
+ 994 │ male
+ 995 │ female
+ 996 │ male
+ 997 │ male
+ 998 │ male
+ 999 │ male
+984 rows omitted
+```
+
+`german[!, Not(Age)]` ! indicates that underlying columns are not copied:
```

Comment:
malformed sentence

Reply:
I did not get this, sorry.

Reply:
1. first run `german[!, Not(Age)]` in your notebook and you will see the problem (in general - do not write anything you have not run)
2. the sentence is not properly formed grammatically. Maybe write something like:

> in the expression `german[!, Not(:Age)]` the `!` character indicates ...

Reply:
Oh yes there has been a typo error. The correct expression is `german[!, Not(:Age)]`. Thanks